canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.scss

```diff
@@ -22,6 +22,20 @@
 		margin-bottom: $spacing-03;
 	}
 	.properties-checkboxset-container {
+		.checkbox-tooltip-container{
+			display: grid;
+			align-items: center;
+			grid-template-columns: 1fr 8fr;
+			width: 100%;
+		}
+		.bx--checkbox-label-text{
```

Comment:
Should we have this margin on the tooltip container instead of a carbon specific class?  That would be less likely to break in the future.